src/components/modal/modal.d.ts

```diff
@@ -1,16 +1,22 @@
 import * as React from "react";
 
 export interface ModalProps {
+  /** Modal content */
+  children?: React.ReactNode;
   /** The ARIA role to be applied to the modal */
   ariaRole?: string;
   /** Determines if the Esc Key closes the modal */
   disableEscKey?: boolean;
+  /** Determines if the Dialog can be closed */
+  disableClose?: boolean;
   /** Determines if the background is disabled when the modal is open */
   enableBackgroundUI?: boolean;
   /** A custom close event handler */
   onCancel?: (ev: React.KeyboardEvent<HTMLElement>) => void;
   /** Sets the open state of the modal */
   open: boolean;
+  /** Transition time */
+  timeout?: number;
 }
 
 declare class Modal<T extends ModalProps> extends React.Component<
```

Comment:
This needs to change as `Modal` is no longer a class anymore - I think it may be the root cause of the `children` problem. 

The `React.Component` type we're extending here has `children` already defined on it I believe so we shouldn't need to specifically add it